code/game/gamemodes/vampire/vampire.dm

```diff
@@ -180,19 +184,14 @@
 	dat += {"To bite someone, target the head and use harm intent with an empty hand. Drink blood to gain new powers.
 You are weak to holy things and starlight. Don't go into space and avoid the Chaplain, the chapel and especially Holy Water."}
 	to_chat(vampire.current, dat)
-	to_chat(vampire.current, "<B>You must complete the following tasks:</B>")
+	to_chat(vampire.current, "<B>The Syndicate will contact you within 10 minutes to give you your objectives. Lay low till then.</B>")
```

Comment:
I don't think vampires get their objectives from the syndicate, unless they are the rare vamp tators.

Reply:
I have just learned that it is actually in the lore that vamps and clings work for the Syndicate. Huh.